java/opendcs/src/main/java/org/opendcs/algorithms/DecayingParameter.java

```diff
@@ -73,7 +70,6 @@ public class DecayingParameter extends AW_AlgorithmBase
 
 	@Output(type = Double.class)
 	public NamedVariable output = new NamedVariable("output", 0);
-	String _outputNames[] = { "output" };
 
 
 	@org.opendcs.annotations.PropertySpec(value = "0.0", propertySpecType = PropertySpec.NUMBER,
```

Comment:
Please try to add an import at the top of the file, and have PropertySpec, instead of org.opendcs.annotations.PropertySpec , for consistency.